src/benchmark/file_io_write_micro_benchmark.cpp

```diff
@@ -72,8 +192,12 @@ BENCHMARK_DEFINE_F(FileIOWriteMicroBenchmarkFixture, IN_MEMORY_WRITE)(benchmark:
 }
 
 // Arguments are file size in MB
-BENCHMARK_REGISTER_F(FileIOWriteMicroBenchmarkFixture, WRITE_NON_ATOMIC)->Arg(10)->Arg(100)->Arg(1000);
-BENCHMARK_REGISTER_F(FileIOWriteMicroBenchmarkFixture, PWRITE_ATOMIC)->Arg(10)->Arg(100)->Arg(1000);
-BENCHMARK_REGISTER_F(FileIOWriteMicroBenchmarkFixture, IN_MEMORY_WRITE)->Arg(10)->Arg(100)->Arg(1000);
+BENCHMARK_REGISTER_F(FileIOWriteMicroBenchmarkFixture, WRITE_NON_ATOMIC_THREADED)
+    ->ArgsProduct({{10, 100, 1000}, {1, 2, 4, 8, 16, 32}})
```

Comment:
is there a reason for the scale of {1, 2, 4, 8, 16, 32}? maybe we should document it

Reply:
not yet, this just basic testing. Would be interesting how many cores our testing system has.